internal/cose_test.go

```diff
@@ -0,0 +1,347 @@
+package internal_test
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"encoding/base64"
+	"testing"
+
+	"github.com/blocky/nitrite/internal"
```

Comment:
Could you separate blocky imports? Here and in `document_test.go`

Reply:
Ah, goland was only sorting "bky.sh" imports. Added `github.com/blocky/` to the ruleset as well